### DISCUSSION TITLE
Switch modes?

### DISCUSSION THREAD

**Discussion by FSHelgeland** [sindrebroch/ha-flexit#29] [Q&A] [answered]
In readme you say that it only polls every 30 minute as default. But if I change mode will it change immediately, as if I use home mode or away mode? Or if I use boost mode as I do use a lot with humidity sensors in the bathroom? Have not got my CL4 yet, and I am wondering if this integration will be good enough for me, or if I should by the modbus-module, and control it through modbus instead. If it changes by request I do not need the modbus, status in HA is not a big problem. Hope you or someone else can answer this. Feel free to write the answer in Norwegian.

Med vennlig hilsen
Freddy

**Comment by sindrebroch** [comment, marked as answer]
Hei! Endring i modus vil skje umiddelbart, men oppdatert status i HA kan ta litt tid. Ved suksessfull endring av status oppdateres modusen optimistisk i HA (altså integrasjonen antar at endringen har tredd i kraft). 

Jeg vil ikke tro at dette er noe stort problem, men jeg skrev kommentaren i tilfelle folk lurer på hvorfor HA i noen tilfeller sier noe annet enn FlexitGO-appen.

**Reply by FSHelgeland** [reply]
Hei Sindre, etter flere lange måneder har jeg nå endelig mottatt anlegget mitt. Har installert anlegg i dag og har nå installert integrasjonen. Får opp alle sensorer (19), og har lagt disse til i et dashboard. Nå tenker jeg lage noe automatisering i NodeRed, men sliter med å finne de ulike modusene jeg skal sette, og også få opp i dashboard hvilken modus som er valgt nå (hjemme,borte,boost, og fire) er vel de jeg trenger å både få aktivert og sett. Noen tips til meg her?

Freddy

**Reply by FSHelgeland** [reply]
Tror jeg fant det ... Har lagt til det siste kortet i dashboard. Tror jeg skal finne ut av det da.

**Reply by FSHelgeland** [reply]
men nei, får skiftet i HA, men hvordan finne preset i NodeRed?

**Reply by FSHelgeland** [reply]
Se bort fra alle disse meldingene, her var det bare litt kunnskap det skortet på. Integrasjonen fungerer helt flott og nå skal jeg få til å lage automasjonene også. Igjen takk for denne fine integrasjonen. 

Freddy

**Comment by FSHelgeland** [comment]
Det høres bra ut da fungerer den vel egentlig som jeg ønsker. Er ikke avhengig av om statusen er korrekt, men fint at anlegget gjør det det skal. Nå bruker jeg heller ikke HA som mitt primære smarthussystem (enda) Da jeg har Sensio, bruker HA til p pushe data til fra Sensio via Node Red og Modbus. Her blir vel jeg vel også å kjøre en del automatisering i Node Red ved avlesing av aquara sine fuktighetssensorer blant annet. Takk for raskt svar, da avventer jeg å bestille modbus-adapter.
